lib/collections/addon/discover/template.hbs

```diff
@@ -1,33 +1,31 @@
-{{title (t 'collections.discover.title')}}
-
 {{discover-page
     query=(action 'query')
     searchResultComponent='collection-search-result'
-    activeFilters=activeFilters
-    consumingService=consumingService
-    discoverHeader=discoverHeader
-    facets=facets
-    lockedParams=lockedParams
+    activeFilters=this.activeFilters
+    consumingService=this.consumingService
+    discoverHeader=this.discoverHeader
+    facets=this.facets
+    lockedParams=this.lockedParams
     
-    queryParams=queryParams
-    searchPlaceholder=searchPlaceholder
+    queryParams=this.queryParams
+    searchPlaceholder=this.searchPlaceholder
     showActiveFilters=true
-    sortOptions=sortOptions
+    sortOptions=this.sortOptions
 
-    sources=sources
-    start=start
-    end=end
-    tags=tags
+    sources=this.sources
+    start=this.start
+    end=this.end
+    tags=this.tags
 
-    page=page
-    provider=provider
-    q=q
-    type=type
-    taxonomy=subject
+    page=this.page
+    provider=this.provider
+    q=this.q
+    type=this.type
+    taxonomy=this.subject
 
-    collectedType=collectedType
-    issue=issue
-    programArea=programArea
-    status=status
-    volume=volume
+    collectedType=this.collectedType
+    issue=this.issue
+    programArea=this.programArea
+    status=this.status
+    volume=this.volume
 }}
```

Comment:
We should go ahead and update this to use angle bracket invocation since we have to modify this anyways.

Reply:
Updated.